lib/Runtime/ByteCode/ByteCodeSerializer.cpp

```diff
@@ -2255,6 +2256,14 @@ class ByteCodeBufferBuilder
             AddDeferredStubs(builder, deferredStubs, function->GetNestedCount(), cache, true);
         }
 
```

Comment:
We actually need ScopeInfo for all (re-)deferrable functions, not just those that are deferred now. You can see in byte code gen how we make the decision to create ScopeInfo based on the FunctionInfo attributes.

Reply:
Got it. I'll check for the CanDefer attribute as well.